src/styles/style.css

```diff
@@ -0,0 +1,163 @@
+.body {
+  margin: 0;
+
+  --border-color: rgb(240, 7, 228);
+  --hover-color: blue;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  max-width: 199px;
+  height: 100%;
+  border: 1px solid var(--border-color);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card__image {
+  box-sizing: border-box;
+  background-image: url(/src/images/imac.jpeg);
```

Comment:
The path for `background-image` URLs should be relative to the CSS file. If `style.css` is in the `styles` directory and the images are in a sibling `images` directory, the correct path would be `../images/imac.jpeg`.